### ISSUE TITLE
Adguard beta for Mac is not working

### ISSUE THREAD

**Issue by maxbat10** [AdguardTeam/AdguardForMac#446]
Adguard Beta 2.0.0.522 does not block any content on my Mac using Safari, Chrome or Firefox.

If I switch to the regular Adguard version 1.5.14, everything directly works well and contents are blocked on Safari, Chrome or Firefox.

Do you have an idea to solve my problem ?

macOS Mojave bêta 10.14.4 (18E220a)
Safari Technology Preview Release 78
Firefox 67.0b4
Chrome 74.0.3729.28

**Comment by ameshkov** [comment]
@maxbat10 

Do you use any network-level software? Antivirus, VPN, firewall, something like this.

**Comment by zzebrum** [comment]
So far I can't repro the issue. 
@maxbat10 When did the problem begin? You had AD v2.0 nightly and after that you've updated to  Mojave beta?

**Comment by maxbat10** [comment]
@ameshkov I use Avira but I tried to disable it and the problem still occurs...

@zzebrum I had Mojave beta with Adguard 1.5.14 and everything was working well. Then I installed Adguard beta and I had the problem. Then if I install Adguard 1.5.14 again, it works well.

Now I use 1.5.14 but I can install the v2 beta again to try to find a solution with your help.

Thank you

**Comment by ameshkov** [comment]
What exact version of Avira? I guess we should also try reproducing this issue with Avira installed.

**Comment by maxbat10** [comment]
My version is 2.0.0.522

**Comment by ameshkov** [comment]
@maxbat10 I mean the version of Avira, not AdGuard

**Comment by maxbat10** [comment]
Oh sorry... I have the Avira 3.10.12.9. It's a beta version but I could try ton install the regular version.

**Comment by zzebrum** [comment]
@maxbat10 ok, I think it's time to ask you to collect debug log :)

Install the AdGuard app v2.0.522, change the logging level in AG settings - advanced to debug, reproduce the problem and send a bug report to support team - in the message mention the link to this thread on GH, please.

**Comment by maxbat10** [comment]
Done !

**Comment by Eugene-Savenko** [comment]
It's in 2150564

**Comment by Stillness-2** [comment]
@maxbat10 This is strange, but according to the log, it seems that more than one `AdGuard` is running at the same time. Can you start the `Activity Monitor` and check that only one `AdGuard` is running under your login?

**Comment by maxbat10** [comment]
No I cas see only one Adguard running under my login.

<img width="842" alt="Screen Activity Monitor" src="https://user-images.githubusercontent.com/11011013/55351589-09b73880-54bf-11e9-86e1-d878f85ecf40.png">

**Comment by maxbat10** [comment]
Any news ?

**Comment by zzebrum** [comment]
@maxbat10 not really. Have you tried to disable Avira? 
we also can arrange teamviewer session if you don't mind - I'll check how it looks from your side. 
or wait a little until next AdGuard and\or Mojave beta will arrive.

**Comment by maxbat10** [comment]
I tried to disable Avira, uninstall Avira 1.5.14 and install Adguard 2.0.0. Now when I launch Adguard beta 2.0, a window asks me to open the System Panel to allow Performiks. But when I open the system panel, nothing appears...
<img width="842" alt="Capture d’écran 2019-04-22 à 20 31 24" src="https://user-images.githubusercontent.com/11011013/56517826-3dc1ce80-653e-11e9-8a75-1d1f5f170fce.png">
<img width="842" alt="Capture d’écran 2019-04-22 à 20 32 25" src="https://user-images.githubusercontent.com/11011013/56517838-42868280-653e-11e9-9c8b-921833050d9d.png">

**Comment by ameshkov** [comment]
@maxbat10 are you on Mojave beta 14.5?

The issue is caused by the changes in the latest Mojave beta, see the details here: #350 

It is resolved in AG for Mac nightly, the beta with the same fix is yet to come.

Meanwhile, here's the direct link to the nightly version: https://agrd.io/mac_nightly

**Comment by maxbat10** [comment]
Everything is ok with Mojave beta 14.5, Adguard nightly version ! Even with Adguard.

Thank you. Hope to have the new beta version soon.

**Comment by zzebrum** [comment]
great, then closing as a duplicate of #350
